src/CordovaNativeSqliteProvider.ts

```diff
@@ -18,6 +18,8 @@ export interface SqlitePluginDbOptionalParams {
     createFromLocation?: number;
     androidDatabaseImplementation?: number;
     androidLockWorkaround?: number;
+    // Database encryption pass phrase
+    key?: string;
```

Comment:
it's called key in sqlite-storage (at least it was so in commneted code)
would prefer to keep the original naming to simplify merge of sqlite-storage encription part.